data/jakarta/test-security-services-proxy-certs.sh

```diff
@@ -124,45 +126,49 @@ else
     echo "Self-signed TLS verification recheck test succeeded"
 fi
 
+# generate CA-signed TLS certificate using openssl
 
+# Generate the Certificate Authority (CA) Private Key
+openssl ecparam -name prime256v1 -genkey -noout -out ca.key
+# Generate the Certificate Authority Certificate
+openssl req -new -x509 -sha256 -key ca.key -out ca.crt -subj "/CN=checkbox-test-ca"
+# Generate the Server Certificate Private Key
+openssl ecparam -name prime256v1 -genkey -noout -out server.key
+# Generate the Server Certificate Signing Request
+openssl req -new -sha256 -key server.key -out server.csr -subj "/CN=localhost"
+# Generate the Server Certificate
+openssl x509 -req -in server.csr -CA ca.crt -CAkey ca.key -CAcreateserial -out server.crt -days 1000 -sha256
```

Comment:
Please move to data/utils.sh and call the function from jakarta and latest suites.

Reply:
pushed new commit with this change